paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentOptionsScreen.kt

```diff
@@ -95,5 +98,8 @@ internal fun PaymentOptionsScreenContent(
                     .padding(horizontal = horizontalPadding),
             )
         }
+
+        Spacer(modifier = Modifier.requiredHeight(bottomPadding))
```

Comment:
I feel like there should be a better way to do this. But I could be wrong, and idk what it is!

To me it seems like it shouldn't care if it's rendered in a frame that's edge to edge or not. The container should handle this portion.

Reply:
Sure. I’m planning to switch from `Column` to `LazyColumn`, which will enable us to set this as the `contentPadding` on the container.